lib/msal-common/test/client/RefreshTokenClient.spec.ts

```diff
@@ -0,0 +1,66 @@
+import chai from "chai";
+import chaiAsPromised from "chai-as-promised";
+import sinon from "sinon";
+import {
+    Authority,
+    AuthorizationCodeClient,
+    Configuration,
+    Constants, RefreshTokenClient, RefreshTokenRequest
+} from "../../src";
+import {
+    AUTHENTICATION_RESULT,
+    DEFAULT_OPENID_CONFIG_RESPONSE,
+    TEST_CONFIG,
+    TEST_TOKENS,
+} from "../utils/StringConstants";
+import {BaseClient} from "../../src/client/BaseClient";
+import {AADServerParamKeys, GrantType} from "../../src/utils/Constants";
+import {ClientTestUtils} from "./ClientTestUtils";
+
+const expect = chai.expect;
+chai.use(chaiAsPromised);
```

Comment:
Can you put this in a central test setup file that is run for all tests?

Reply:
Split it into it's own PR: https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/1527